bin/run-tests.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
+
+# Synopsis:
+# Test the test runner by running it against a predefined set of solutions
+# with an expected output.
+
+# Output:
+# Outputs the diff of the expected test results against the actual test results
+# generated by the test runner.
+
+# Example:
+# ./bin/run-tests.sh
+
+exit_code=0
+
+# Iterate over all test directories
+for test_dir in tests/*; do
+    test_dir_name=$(basename "${test_dir}")
+    test_dir_path=$(realpath "${test_dir}")
+    results_file_path="${test_dir_path}/results.json"
+    expected_results_file_path="${test_dir_path}/expected_results.json"
+
+    bin/run.sh "${test_dir_name}" "${test_dir_path}" "${test_dir_path}"
+
+    # OPTIONAL: Normalize the results file
+    # If the results.json file contains information that changes between
+    # different test runs (e.g. timing information or paths), you should normalize
+    # the results file to allow the diff comparison below to work as expected
+    # sed -i -E \
+    #   -e 's/Elapsed time: [0-9]+\.[0-9]+ seconds//g' \
+    #   -e "s~${test_dir_path}~/solution~g" \
+    #   "${results_file_path}"
```

Comment:
Feel free to remove this comment.

Reply:
removed in https://github.com/exercism/abap-test-runner/pull/7/commits/c27b70d1104ff6dd5500d331067df51214cc5a77